src/django_simple_nav/permissions.py

```diff
@@ -30,14 +30,14 @@ def check_item_permissions(item: NavGroup | NavItem, request: HttpRequest) -> bo
         )
         return True
 
+    if not hasattr(request, "user"):
+        # if no user attached to request, we assume that the user is not authenticated
+        # and we should hide if *any* permissions are set
+        return not item.permissions
+
     user = cast(User, request.user)
```

Comment:
> The cast needed to make static type checkers happy fails if django.contrib.auth is installed but there is no user attached to a request.

This might be worth a comment/note here.